wallet/src/main/scala/org/bitcoins/wallet/api/WalletApi.scala

```diff
@@ -109,6 +111,7 @@ trait LockedWalletApi extends WalletApi {
   /**
     * Mimics the `getaddressinfo` RPC call in Bitcoin Core
     *
+    * @param address
```

Comment:
No point in having just a `@param` annotation if there's no description of the parameter